pillarbox-core-business/src/main/java/ch/srgssr/pillarbox/core/business/images/DefaultImageScaleService.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023. SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.core.business.images
+
+import ch.srgssr.pillarbox.core.business.images.ImageScaleService.ImageFormat
+import ch.srgssr.pillarbox.core.business.images.ImageScaleService.ImageWidth
+import ch.srgssr.pillarbox.core.business.integrationlayer.service.IlHost
+import java.net.URL
+
+/**
+ * Default service for scaling images. This only works for SRG images.
+ *
+ * @property baseUrl Base URL of the service.
+ */
+class DefaultImageScaleService(
+    private val baseUrl: URL = IlHost.DEFAULT
+) : ImageScaleService {
+    override fun getScaledImageUrl(
+        imageUrl: String,
+        width: ImageWidth,
+        format: ImageFormat
+    ): String {
+        return "${baseUrl}images/?imageUrl=$imageUrl&format=${format.format}&width=${width.width}"
```

Comment:
Don't you have a safe way to build URLs instead of manually appending parameters?

Reply:
Indeed, I've changed it in https://github.com/SRGSSR/pillarbox-android/pull/290/commits/f9b685fb936346b0643e986abdd85e563c007176